generate/adapter_v2.py

```diff
@@ -84,10 +84,17 @@ def main(
     prompt_length = encoded.size(0)
 
     t0 = time.perf_counter()
-    y = generate(model, encoded, max_new_tokens, temperature=temperature, top_k=top_k, eos_id=tokenizer.eos_id)
+    y = generate(
+        model,
+        idx=encoded,
+        max_seq_length=max_new_tokens,
```

Comment:
What's the reasoning behind this?

Reply:
Using named arguments for easier debugging I guess

Reply:
I mean passing `max_seq_length=max_new_tokens`. This would limit it a lot as by default it will equal to the `block_size`

Reply:
I don't know, to be honest. I adopted this from the regular LLaMA adapter in the [adapter.py](https://github.com/Lightning-AI/lit-llama/blob/main/generate/adapter.py#L85) script when I originally implemented adapter_v2.py. It's also like this in `generate/lora.py`

I'd say it's okay to leave this for this PR, but then we maybe want to open an issue/PR to revisit this for ALL generate scripts?

Reply:
The links you shared are doing it as I'm suggesting, to be clear, this is what I mean

```suggestion
```